content/en/scale-agent/reference/api/dynamic-accounts.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Dynamic Accounts API Developer Guide
+linkTitle: Dynamic Accounts API
+description: >
+   Learn about the Clouddriver HTTP REST API Dynamic Accounts endpoints for the Armory Scale Agent for Spinnaker and Kubernetes.
+---
+
+## Get Armory accounts
+`GET /armory/accounts/{accountId}`
+
+Return a list of all managed accounts, including static accounts defined in agent config files, and dynamic accounts defined in Clouddriver. The returned list is sorted by account name in ascending order.
+
+### Request parameters
+ - `page` -  The page number for paginating results, starting from 1. This parameter defaults to 1 if omitted.
+ - `limit` How many accounts to return. Defaults to 100 if omitted.
+
+### Example response
+``` json
+{
+  "items": [
+   {example of list items needed here}
+  ]
+  "page": 1,
+  "limit": 3,
+  "total": 542
+}
+```
+## Get an account by ID
+
+`GET /armory/accounts/{accountId}`
+
+Return account details based on the account ID.
+
+### Top level response fields
+ - `name`: The account name.
+ - `type`: The cloud provider name (currently always kubernetes).
+ - `source`: The account origin - either `clouddriver` or  `static-agent-config`.
+ - `config`: The account configuration as defined in the underlying data stores.
+ - `zoneId`: Identifier of the zone to which this account is pinned (optional). A missing value means that the account can be dynamically assigned to any available agent.
+ - `agents`: The List of agent identifiers confirmed to handle the account, either for making deployments, watching cluster events, or both.
+ - `lastAssignmnentMsg`: This string represents the last message produced when assigning an account to an agent. It is used to reveal communication errors, for example using an invalid `kubeconfig` file. 
+
+### Example response
+``` json
+{
+   "name":"my-account-1",
+   "type":"kubernetes",
+   "source":"clouddriver",
+   "config":{
+
+   },
+   "zoneId":"agent-private-network-1",
+   "agents":{
+      {
+         "agentId":"agent-private-network-1-54865d798c-cpqgm",
+         "caching":true
+      },
+      {
+         "agentId":"agent-private-network-1-54865d798c-tdfvw",
+         "caching":false
+      }
+   },
+   "lastAssignmentMsg":"successfully assigned to agent agent-private-network-1-54865d798c-tdfvw for executing operations"
+}
+```
+## Migrate Clouddriver account to Agent
+`POST /armory/accounts`
+
+Migrate an existing native clouddriver account an agent.
+The `zoneId` is an optional parameter. If present, the account is pinned to agents matching the same `zoneId` in the Agent configuration. If omitted, the account is assigned to any available Agent pod. 
```

Comment:
This is correct, but I would add "same `zoneId` in the Agent configuration or its computed value deploymentName_namespace". 

Let's also add the rest of the attributes as defined in [dynamic accounts](https://armory.slab.com/posts/k-8-s-agent-dynamic-db-accounts-usage-guidelines-i9indwic)